src/screens/VisualTests/VisualTests.tsx

```diff
@@ -157,6 +158,7 @@ interface VisualTestsProps {
   setAccessToken: (accessToken: string | null) => void;
   setIsOutdated: (isOutdated: boolean) => void;
   setIsRunning: (isRunning: boolean) => void;
+  updateBuildStatus: (build: StartedBuild | CompletedBuild) => void;
   storyId: string;
```

Comment:
It seems these types are defined multiple times, and should likely be shared/extracted.

Reply:
Do you have a suggestion on how to do that?